analyzer/src/functTest/java/org/jboss/gm/analyzer/alignment/OpenTelemetryFunctionalTest.java

```diff
@@ -94,6 +94,7 @@ public void verifyOpenTelemetryGradle() throws IOException, URISyntaxException,
     public void verifyOpenTelemetryKotlin() throws IOException, URISyntaxException, ManipulationException {
         // XXX: Use of pluginManagement.plugins{}
         assumeTrue(GradleVersion.current().compareTo(GradleVersion.version("5.6")) >= 0);
+        assumeTrue(GradleVersion.current().compareTo(GradleVersion.version("7.4.2")) <= 0);
```

Comment:
Should be `7.5 < 0` in case there's a 7.4.3?

Reply:
Good point, thanks.